src/test/java/com/xceptance/neodymium/testclasses/data/annotation/inheritance/ChildInheritingDtoFromAnnotation.java

```diff
@@ -0,0 +1,16 @@
+package com.xceptance.neodymium.testclasses.data.annotation.inheritance;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.xceptance.neodymium.util.DataUtils;
+
+public class ChildInheritingDtoFromAnnotation extends ParentClassWithDtoFromAnnotation
+{
+    @Test
```

Comment:
You can use the new feature to optimize your code :-)
Just instantiate the testId as field and use it within the functions.

```
    @Data("$.testId")
    private String testId;
```
after the change above should the following work:
```
    @Data
    private String testId;
```

Reply:
Btw. Please add this change to all test classes of this feature if it optimizes the code.

Reply:
Done